examples_server/examples/data_table/large_table_column_reordering/data_table.js

```diff
@@ -0,0 +1,125 @@
+Backdraft.app("TableExample", {
+
+  plugins : [ "DataTable"],
```

Comment:
Styleguide calls for no spaces between key name and `:`, looks like all keys in this file have a single space.